tests/ecbuild_install_dependency_with_soversion/run-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -ex
+
+echo "HI"
+
+TEST_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+foo_src="$TEST_ROOT/libfoo"
+project_src="$TEST_ROOT/project"
+
+mkdir -p foo-{build,install}
+cmake -B foo-build -S "$foo_src" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="$(pwd)/foo-install"
+cmake --build foo-build --target install -j
+
+mkdir -p project-{build,install}
+cmake -B project-build -S "$project_src" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="$(pwd)/project-install" -DCMAKE_PREFIX_PATH="$(pwd)/foo-install" -Decbuild_ROOT="$TEST_ROOT/../../"
+cmake --build project-build --target install -j
+
```

Comment:
Could we add here trying to run the executable?  No "Library not found" runtime error should appear.
```
$(pwd)/project-install/bin/project
```

Reply:
ok